--- error.expected	Sat Mar 31 16:15:31 2001
+++ error.output	Thu Apr 19 23:47:53 2001
@@ -1,5 +1,5 @@
 SELECT invalid_type_uncaught('rick');
-ERROR:  plpython: Call of function `__plpython_procedure_invalid_type_uncaught_1175341' failed.
+ERROR:  plpython: Call of function `__plpython_procedure_invalid_type_uncaught_1289666' failed.
 plpy.SPIError: Cache lookup for type `test' failed.
 SELECT invalid_type_caught('rick');
 NOTICE:  ("Cache lookup for type `test' failed.",)
@@ -9,7 +9,7 @@
 (1 row)
 
 SELECT invalid_type_reraised('rick');
-ERROR:  plpython: Call of function `__plpython_procedure_invalid_type_reraised_1175343' failed.
+ERROR:  plpython: Call of function `__plpython_procedure_invalid_type_reraised_1289668' failed.
 plpy.Error: ("Cache lookup for type `test' failed.",)
 SELECT valid_type('rick');
  valid_type 
